README.md

```diff
@@ -85,6 +86,23 @@ will turn on the `no-unnamed-features` rule.
 
 Any tag not included in this list won't be allowed.
 
+### file-name
+
+`file-name` is configured with a style to enforce. The default is `TitleCase`:
+
+```json
+{
+  "file-name": ["on", {"style": "TitleCase"}]
+}
+```
+
+The list of supported styles is:
+
+- `TitleCase`
```

Comment:
I think that `TitleCase` refers to strings where each word starts with a capital letter and spaces between words are maintained. 
Sources: 
- https://wiki.c2.com/?CapitalizationRules
- https://en.wikipedia.org/wiki/Camel_case (3d paragraph)
`Title Case` maps exactly to the output of `_.startCase()`


When describing strings where each word starts with a capital letter and there are no spaces between words, the term I'm familiar with is `PascalCase`. 

Would you mind using that terminology instead? I think we could support both `Title Case` and `PascalCase`

Reply:
Okay makes sense - done